src/Test/Functional/Acceptance/SessionConfigurationCest.php

```diff
@@ -0,0 +1,90 @@
+<?php
+//Zephyr ID MAGECLOUD-46
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MagentoCloud\Test\Functional\Acceptance;
+
+use Magento\MagentoCloud\Test\Functional\Codeception\Docker;
+
+/**
+ * This test runs on the latest version of PHP
```

Comment:
This description is meaningless,s hould be replaced on test description

Reply:
Done